manifests/init.pp

```diff
@@ -445,153 +445,130 @@
 #   Default: 'Sensu'
 
 class sensu (
-  $version                        = 'installed',
-  $sensu_plugin_name              = 'sensu-plugin',
-  $sensu_plugin_provider          = $::osfamily ? {
+  Pattern[/^absent$/, /^installed$/, /^latest$/, /^present$/, /^[\d\.\-el]+$/] $version = 'installed',
```

Comment:
Some tests might fail, and this is probably the same validation which was there before